### ISSUE TITLE
Result type of stopovers

### ISSUE THREAD

**Issue by ialokim** [public-transport/fpti-js#4]
As stated [in the documentation](/public-transport/fpti-js/blob/master/docs/stopovers.md), the `stopovers` method returns a `Promise` resulting in an `Array` of `stopover` objects as [defined by `FTPF`](/public-transport/friendly-public-transport-format/blob/master/spec/readme.md#stopover).

I'm not sure if I'm missing something obvious, but this object does only contain information for time and platform (optional), but not for the line nor the station the public transport service is heading to.

How should this information be added to the resulting array?

**Comment by juliuste** [comment]
You're right, even though I'd say that we could rather add `line` as an optional attribute in FPTF, @derhuerst?

**Comment by ialokim** [comment]
> add `line` as an optional attribute for `stopover` in FPTF

Sounds reasonable to me, but `line` seems to require (?) a `routes` array, which would not make too much sense for the `stopovers` api, and it does not expose any sort of "direction" key really necessary for `stopovers`.

**Comment by derhuerst** [comment]
> we could rather add `line` as an optional attribute for `stopover`

I'd rather add a `departure`/`arrival` type that has a `stopover` and a `line`.

**Comment by ialokim** [comment]
> I'd rather add a departure/arrival type that has a stopover and a line.

... and two keys `origin` and `destination` to express the station where the public transport service is coming from (arrival) or heading to (departure)?
